src/server.cpp

```diff
@@ -1676,7 +1678,8 @@ void Server::SendAddParticleSpawner(session_t peer_id, u16 protocol_version,
 
 	pkt.putLongString(texture);
 
-	pkt << id << vertical;
+	pkt << id;
+	pkt << true; // Dummy value for vertical
```

Comment:
since the default is `vertical = false`, it should probably be used here instead (since old clients will still interpret this value)

Reply:
You're right, thanks